packages/fetchai/skills/tac_control/game.py

```diff
@@ -445,15 +445,16 @@ def _check_consistency(self) -> None:
                 self.counterparty_amount >= 0
             ), "Counterparty_amount must be positive when the counterpary is the payment receiver."
 
-    def has_matching_signatures(self, api: LedgerApi) -> bool:
```

Comment:
we want to avoid depending on the full ledger api here as we don't need the remote functionalities